### PR TITLE
rbd-fuse: implement mv operation

### PR DESCRIPTION
**Pull request by wuxiangwei**
implement mv operation for rbd-fuse to rename image.

Signed-off-by: Xiangwei Wu wuxiangwei@h3c.com

### CODE REVIEW

**Comment by dillaman** [comment]
@wuxiangwei Can you add some validation for the destination image name?  It shouldn't be blank and shouldn't contain "/" nor "@" characters.

**Comment by wuxiangwei** [comment]
@dillaman i have add validation for destination image name, please take a look. thank you.

File: src/rbd_fuse/rbd-fuse.cc [hunk 1]
```diff
@@ -534,6 +535,31 @@ rbdfs_create(const char *path, mode_t mode, struct fuse_file_info *fi)
 	return r;
 }
 
+int rbdfs_rename(const char *path, const char *destname)
+{
+    const char *srcname = NULL;
+    const char *extra[] = {" ", "@", "/"};
```
Comment (dillaman) [inline src/rbd_fuse/rbd-fuse.cc]: I don't think there is any restriction for spaces in the image name -- at least the rbd CLI doesn't restrict it.

**Comment by wuxiangwei** [comment]
@dillaman hi, i have remove the restriction for spaces in the image name , please check again, thank you .

**Pull request merged**